### DISCUSSION TITLE
[Feature] Open image in Immich app

### DISCUSSION THREAD

**Discussion by OverHash** [immich-app/immich#7363] [Feature Request]
### The feature

When going through my phones file explorer, if there is an image I want to view I would love to use Immich as the place to open and view this image.

Currently, I am forced to use the native gallery. Apps like Google Photos will allow you to open photos (that are even not part of the gallery) to view, and provide a way to choose to upload the photo to the Google Photos gallery. I would love for Immich to have this.

Related discussions include #1641 (using Immich to select photos to share) and #2659 (which discusses "sharing" to Immich, though I'm not entirely clear what the flow is).

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by AZweimiller** [comment]
This feels like a dupe of https://github.com/immich-app/immich/discussions/1605
